code/Rounds/PreRound.cs

```diff
@@ -18,5 +18,21 @@ public override void OnPlayerSpawn(MainPlayer player)
             base.OnPlayerSpawn(player);
             AddPlayer(player);
         }
+
+        protected override void OnTimeUp()
+        {
+            Log.Info("Times up");
+            AssignRoles();
+            base.OnTimeUp();
```

Comment:
dans ce cas de figure le comportement de base devrait logiquement être avant non ?
```suggestion
            base.OnTimeUp();
            Log.Info("Times up");
            AssignRoles();
```

Reply:
Je sais pas mais je dois delete le loginfo

Reply:
```suggestion
            base.OnTimeUp();
            AssignRoles();
```